samples/bluetooth/mesh/light/boards/thingy53_nrf5340_cpuapp.conf

```diff
@@ -6,56 +6,15 @@
 ################################################################################
 # Application overlay - Thingy:53 DFU
 
-CONFIG_MCUMGR=y
-CONFIG_MCUMGR_SMP_BT=y
-CONFIG_MCUMGR_SMP_BT_AUTHEN=n
-CONFIG_MCUMGR_SMP_BT_CONN_PARAM_CONTROL=y
-
-CONFIG_MCUMGR_CMD_IMG_MGMT=y
-CONFIG_MCUMGR_CMD_OS_MGMT=y
-
-# Enable MCUmgr Packet Reassembly feature over Bluetooth and its
-# configuration dependencies. MCUmgr buffer size is optimized to fit one SMP
-# packet divided into five Bluetooth Write Commands, transmitted with the
-# maximum possible MTU value: 498 bytes.
-CONFIG_MCUMGR_SMP_REASSEMBLY_BT=y
-CONFIG_MCUMGR_BUF_SIZE=2475
-CONFIG_OS_MGMT_MCUMGR_PARAMS=y
-
-# Enable custom SMP request to erase settings partition.
-CONFIG_MCUMGR_GRP_ZEPHYR_BASIC=y
-CONFIG_MCUMGR_GRP_BASIC_CMD_STORAGE_ERASE=y
+CONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU=y
+CONFIG_MCUBOOT_IMAGE_VERSION="2.0.0+0"
 
-# Increase system workqueue stack size to prevent stack overflows while handling
-# SMP requests.
 CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=4096
 
-CONFIG_BT_L2CAP_TX_MTU=498
```

Comment:
Sample's configuration file sets `CONFIG_BT_L2CAP_TX_MTU=69` which actually affects the generated configuration.
I think we need to override it here. Please check also other samples (e.g. compare `.config` files before and after your change).

Reply:
Maybe we could consider emitting some sort of warning to ensure that user is aware of the fact that some default values were modified and configuration is not correct? We could discuss this on Monday.

Reply:
> Sample's configuration file sets `CONFIG_BT_L2CAP_TX_MTU=69` which actually affects the generated configuration. I think we need to override it here. Please check also other samples (e.g. compare `.config` files before and after your change).

Thanks, I corrected it also in other Mesh samples as they were also affected

Reply:
> Maybe we could consider emitting some sort of warning to ensure that user is aware of the fact that some default values were modified and configuration may not be correct? We could discuss this improvement on Monday.

I would leave it out of the scope of this PR